jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/surfaces/AppBar.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.jdi.light.material.elements.surfaces;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.material.asserts.surfaces.AppBarAssert;
+import org.openqa.selenium.By;
+
+public class AppBar extends UIBaseElement<AppBarAssert> {
```

Comment:
For App bar I'd like to add following methods:
1.	Method for open menu
2.	Method to get button by index (this method should return Button instance)
3.	Method to get button by text (this method should return Button instance)
4.	Method to get title text

Please delete all unused methods